sql/catalog/lease: TestTableCreationPushesTxnsInRecentPast failed

sql/catalog/lease.TestTableCreationPushesTxnsInRecentPast [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20727891?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20727891?buildTab=artifacts#/) on master @ [b6a0efc2d0a69c31b76ea7083e3fc3e71b2cee01](https://github.com/cockroachdb/cockroach/commits/b6a0efc2d0a69c31b76ea7083e3fc3e71b2cee01):


```
=== RUN   TestTableCreationPushesTxnsInRecentPast
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/ede32a302f54488286767858cc25fd06/logTestTableCreationPushesTxnsInRecentPast3825064928
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:155: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    lease_test.go:1818: 
        	Error Trace:	pkg/sql/catalog/lease/lease_test.go:1818
        	Error:      	Received unexpected error:
        	            	pq: relation "t.kv" does not exist
        	Test:       	TestTableCreationPushesTxnsInRecentPast
    panic.go:615: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/ede32a302f54488286767858cc25fd06/logTestTableCreationPushesTxnsInRecentPast3825064928
--- FAIL: TestTableCreationPushesTxnsInRecentPast (4.15s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTableCreationPushesTxnsInRecentPast.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-56435

--- a/pkg/sql/catalog/lease/lease_test.go
+++ b/pkg/sql/catalog/lease/lease_test.go
@@ -1768,7 +1768,7 @@ func TestTableCreationPushesTxnsInRecentPast(t *testing.T) {
 		ServerArgs: base.TestServerArgs{
 			Knobs: base.TestingKnobs{
 				Store: &kvserver.StoreTestingKnobs{
-					MaxOffset: time.Second,
+					MaxOffset: 2 * time.Second,
 				},
 			},
 		},

